backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Models/ReceivedDelegation.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Text.Json.Serialization;
+using Altinn.AuthorizationAdmin.Core.Models.ResourceRegistry;
+
+namespace Altinn.AuthorizationAdmin.Core.Models
+{
+    /// <summary>
+    /// This model describes a delegation. A delegation is an action that says if a resource is delegated to you or you have delegated a resource to another person/org
+    /// </summary>
+    public class ReceivedDelegation
+    {
+        /// <summary>
+        /// Gets or sets the identifier of the resource in a delegation
+        /// </summary>
+        [JsonPropertyName("reporteename")]
+        public string ReporteeName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the resource in a delegation
+        /// </summary>
+        [JsonPropertyName("offeredbypartyid")]
+        public int OfferedByPartyId { get; set; }
+
+        /// <summary>
+        /// Resources that were received
+        /// </summary>
+        public List<ServiceResource> Resources { get; set; }   
```

Comment:
Listing the entire ServiceResource model might be a bit overkill for what the function is to be used for?
But organization number for the OfferedBy might be nice.

Reply:
Documentation summary for ReporteeName and OfferedByPartyId needs an update

Reply:
ok, I was working on this now. Let me then push only the other changes